src/components/CheckboxField/__stories__/index.stories.tsx

```diff
@@ -28,12 +28,16 @@ Default.args = {
   name: 'default',
 }
 
-export const Checked = Template.bind({})
-
-Checked.args = {
-  checked: true,
-  name: 'checked',
-}
+export const Checked: Story<FormProps> = ({ errors }) => (
```

Comment:
Can we have an example with one checked with boolean and another example with multi initial selected ?